warmup.krun

```diff
@@ -166,12 +173,20 @@ else:
 
 # Copy off results after each execution -- soft-dev specific!
 #
+# This assumes a no-password SSH key named 'id_rsa' is present in this
+# directory, and that the corresponding public key is installed on the correct
+# user on the remote machine.
+#
 # We allow failure, otherwise Krun will halt the experiment if (e.g.) the
 # SSH server hostname is temporarily unavailable.
 HOSTNAME = platform.node().split(".")[0]
-SCP_CMD = ("tar czf - ${KRUN_RESULTS_FILE} ${KRUN_LOG_FILE} | "
-           "ssh -i id_rsa " "vext01@bencher2.soft-dev.org "
-           "'cat > research/krun_results/%s.tgz'" % HOSTNAME)
+REMOTE_LOGIN = "vext01@bencher2.soft-dev.org"
```

Comment:
Sarah's right, it is a bit weird to have these things in here... or, at least, to have them lurking a bit randomly later in the file. Maybe we can bring all the "softdev specific variables" to the top of the file, so that other people stand a chance of running this on their machines?